code/game/machinery/computer/arena.dm

```diff
@@ -367,6 +583,7 @@
 	var/datum/browser/popup = new(user, "arena controller", "Arena Controller", 500, 600)
 	popup.set_content(dat.Join())
 	popup.open()
+	*/
 
 /// Arena spawnpoint
 /obj/machinery/arena_spawn
```

Comment:
One thing to consider, spawning may be loaded into the map directly so it will be different than last event when we loaded people into static zones that didn't change with arena maps. I'm not sure if the previous code was set up to account for that though.